hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/ContainerManagerImpl.java

```diff
@@ -279,6 +257,24 @@ public void notifyContainerReportProcessing(final boolean isFullReport,
     throw new UnsupportedOperationException("Not yet implemented!");
   }
 
+  @Override
+  public void deleteContainer(final ContainerID containerID)
+      throws IOException {
+    final HddsProtos.ContainerID id = containerID.getProtobuf();
```

Comment:
Need to hold the writeLock?

Reply:
Thanks for the review. Missed this somehow 😄